### ISSUE TITLE
Reach Router support

### ISSUE THREAD

**Issue by danielnixon** [oaf-project/oaf-routing#8]
https://reach.tech/router

**Comment by danielnixon** [comment]
Reach Router takes care of focus already but doesn't announce navigation to screen readers.

This will likely require a new setting to _not_ manage focus (allowing Reach Router to keep doing its thing) while we make the screen reader announcements.

**Comment by danielnixon** [comment]
https://github.com/oaf-project/oaf-routing/commit/f64fc4a5d5fec809f0d4be62508e2cd6f78a91cb

**Comment by danielnixon** [comment]
https://github.com/oaf-project/oaf-reach-router
